aiohttp/http_websocket.py

```diff
@@ -85,6 +85,12 @@ class WSMsgType(IntEnum):
     ERROR = 0x102
 
 
+MESSAGE_TYPES_WITH_CONTENT: Final = (
+    WSMsgType.BINARY,
+    WSMsgType.TEXT,
+    WSMsgType.CONTINUATION,
+)
```

Comment:
Should be a frozenset if we're using `in` below?

Reply:
The performance won't make a difference until we get to more than a few items.

Reply:
```
0.013342750000447268
0.018335040998863406
```
frozenset is tiny bit faster

Reply:
https://github.com/aio-libs/aiohttp/pull/8747

Reply:
Yeah, tuples are only slightly faster when the first item matches and slower the rest.of the time, so good practice to just always use sets.